.env.example

```diff
@@ -7,4 +7,8 @@ DATABASE_URL=postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@db:${DATABASE_PO
 
 SECRET_KEY=
 ALGORITHM=
-ACCESS_TOKEN_EXPIRE_MINUTES=
\ No newline at end of file
+ACCESS_TOKEN_EXPIRE_TIME=
+
+# Configuration to run migration
+ALEMBIC_DB_PORT=
+ALEMBIC_DB_HOST=
```

Comment:
@gustavosmonteiro after this PR is merged you will need to update your .env file with these variables.

Reply:
@gustavosmonteiro you also need to change the variable `ACCESS_TOKEN_EXPIRE_MINUTES` to `ACCESS_TOKEN_EXPIRE_TIME`